modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -532,12 +532,29 @@ definition {
 	}
 
 	@description = "Verify that the user can select Email as Notification type"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanSelectEmailAsNotificationType {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanSelectEmailAsNotificationType";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.selectNode(nodeType = "start");
 
-		// TODO LRQA-73842 CanSelectEmailAsNotificationType pending implementation
+		ProcessBuilderKaleoDesignerReact.viewAndFillNotification(
+			name = "New Notification",
+			newNotification = "true",
+			notificationTemplateValue = "New Notification Template",
+			notificationTypeOption = "email");
+
+		ProcessBuilderKaleoDesignerReact.backArrowOnSidebar();
+
+		ProcessBuilderKaleoDesignerReact.gotoNotification(notificationName = "New Notification");
 
+		ProcessBuilderKaleoDesignerReact.viewNotificationInfo(
+			checkboxValue = "select",
```

Comment:
This checkboxValue variable it's not present at the macro, the cehckboxes are already configured depending on the option you choose, why you have used this? If you have any questions about variables or macros please reach me on slack or in person. Thanks.